helm/mojaloop-barebone/templates/elephant-connector-mojaloop-java-ingress.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: in02tn04-ph-ee-connector-mojaloop-java
+  annotations:
+    kubernetes.io/ingress.class: azure/application-gateway
+spec:
```

Comment:
you can disable all ingresses

Reply:
It's a mojaloop chart. No need to change anything with this paymenthub barebone PR.